src/error.rs

```diff
@@ -41,3 +45,14 @@ pub enum Error {
     #[error("The value {0} in {1} cannot be represented as type {2}")]
     TypeCast(String, String, &'static str),
 }
+
+#[cfg(feature = "ordered-float")]
+impl From<ordered_float::ParseNotNanError<ParseFloatError>> for Error {
+    fn from(err: ordered_float::ParseNotNanError<ParseFloatError>) -> Self {
```

Comment:
I wonder whether `ParseFloatError` should/could be generic instead. Not sure.

Reply:
Implementing generics for just the standard types was a huge PITA, so I wouldn't bother for more obscure stuff.